ios/PhyWeb/UI/PWBeaconsViewController.m

```diff
@@ -329,6 +329,21 @@ - (void)_updateBeaconsNow {
 - (void)_sort {
   BOOL stableMode = [[PWBeaconManager sharedManager] isStableMode];
 
+  BOOL hasRank = NO;
+  for(PWBeacon * beacon in _beacons) {
+    if ([beacon hasRank]) {
+      hasRank = YES;
+      break;
+    }
+  }
+  if (hasRank) {
+    for(PWBeacon * beacon in _beacons) {
+      if (![beacon hasRank]) {
+        [beacon setHasRank:YES];
```

Comment:
We do the same in the Android client: https://github.com/google/physical-web/pull/277/files

From what I understood it was supposed to be used while we transitioned to server side ranking.